external/storm-hdfs/src/test/java/org/apache/storm/hdfs/spout/TestHdfsSpout.java

```diff
@@ -203,15 +209,13 @@ public void testEmptySimpleText_ACK() throws Exception {
             conf.put(Config.TOPOLOGY_ACKER_EXECUTORS, "1"); // enable ACKing
             openSpout(spout, 0, conf);
 
-            // consume empty file
-            runSpout(spout, "r1");
-            Path arc1 = new Path(archive.toString() + "/file_empty.txt");
```

Comment:
I believe this test was added after I accidentally turned off tests. It has never worked. The comments below should describe what the test is doing, and how we want the spout to behave.